### PR TITLE
Add prefix reset command

### PR DESCRIPTION
**Pull request by Erik1000**

### CODE REVIEW

File: src/prefixes.rs [hunk 1]
```diff
@@ -93,6 +93,27 @@ pub async fn prefix_remove(
...
+        .await?
+        .rows_affected();
+
+    let msg = if prefixes_deleted == 0 {
+        "You have no prefixes to deleted".to_string()
+    } else if prefixes_deleted == 1 {
+        "1 prefix was deleted".to_string()
+    } else {
+        format!("{} prefixes were deleted", prefixes_deleted)
+    };
```
Comment (Erik1000) [inline src/prefixes.rs:111]: I know that's ugly. Should I, as an alternative, just not display the number of prefixes removed? Or use the classical `prefix(es)`? Or what's the way you'd suggest?

Comment (kangalioo) [reply src/prefixes.rs:111]: Nah it's fine. It's less ugly than the classical `prefix(es)` spelling in my opinion :p

Btw it should be "You have no prefixes to **delete**"

Comment (Erik1000) [reply src/prefixes.rs:111]: I'll fix that typo and run cargo-sqlx prepare. give me a minute

**Comment by kangalioo** [comment]
Thanks!

**Pull request merged**